src/modules/facility-loan-transaction/facility-loan-transaction.controller.test.ts

```diff
@@ -0,0 +1,50 @@
+import { PROPERTIES } from '@ukef/constants';
+import { DateStringTransformations } from '@ukef/modules/date/date-string.transformations';
+import { GetFacilityLoanTransactionGenerator } from '@ukef-test/support/generator/get-facility-loan-transaction-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { when } from 'jest-when';
+
+import { FacilityLoanTransactionController } from './facility-loan-transaction.controller';
+import { FacilityLoanTransactionService } from './facility-loan-transaction.service';
+
+jest.mock('./facility-loan-transaction.service');
+
+describe('FacilityLoanTransactionController', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const dateStringTransformations = new DateStringTransformations();
+  const portfolioIdentifier = PROPERTIES.GLOBAL.portfolioIdentifier;
+  const facilityIdentifier = valueGenerator.ukefId();
+  const bundleIdentifier = valueGenerator.string({ length: 9 });
```

Comment:
I checked dev and prod. Looks like bundleIdentifier is always 10 digits, maybe I said that is 9 before.

Reply:
Changed to use `acbsBundleId` method.